app/src/main/java/org/wikipedia/bridge/CommunicationBridge.java

```diff
@@ -70,7 +71,7 @@ public void onPageFinished() {
     public void resetHtml(@NonNull String wikiUrl, String title) {
         isDOMReady = false;
         pendingJSMessages.clear();
-        communicationBridgeListener.getWebView().loadUrl(wikiUrl + RestService.REST_API_PREFIX + RestService.PAGE_HTML_ENDPOINT + UriUtil.encodeURL(title));
+        communicationBridgeListener.getWebView().loadUrl(wikiUrl + RestService.REST_API_PREFIX + RestService.PAGE_HTML_ENDPOINT + UriUtil.encodeURL(StringUtil.addUnderscores(title)));
```

Comment:
Could we change the `title` parameter to be a `PageTitle`, and then use `title.getPrefixedText()` here?

Reply:
Yes! It makes sense. But I would use `title.getConvertedText()` to prevent the variant issue